### DISCUSSION TITLE
New Graphics3D, prototype for Manipulate and JS output cells

### DISCUSSION THREAD

**Discussion by JerryI** [WLJSTeam/wljs-notebook#16] [Documentation]
Some new features
PS: Sorry, my gif compressor is bad, when the updates of a picture a small. It the reality it looks 2x faster

### early prototype for `Manipulate` function using `FrontEndExecutable` with reactive chain update
![Screen Recording 2023-03-15 at 10 44 20](https://user-images.githubusercontent.com/4111822/226285017-34f3928c-6dbf-404b-9eb9-58cdbf9fe4f6.gif)

### water, sky shared. `Roughness[]`, `Metallness[]`, `Emissive[]` parameters for the materials
![Screen Recording 2023-03-14 at 22 44 55](https://user-images.githubusercontent.com/4111822/226285077-9a70b323-659a-44a8-acca-f630dd65fe19.gif)

### alternative method of handling animations using `RequestAnimationFrame`
![Screen Recording 2023-03-15 at 10 41 47](https://user-images.githubusercontent.com/4111822/226285104-9f0c9b76-749e-4eaa-ab4a-e384c6b42ca9.gif)

### direct communication between JS cell and Wolfram Language. Real-time Many-body modeling
![boxes-trimmed](https://user-images.githubusercontent.com/4111822/226285168-12f65efa-57ef-45d2-ba09-51df2ed894e9.gif)

**Comment by userrand** [comment]
Looks very nice 🎉 !
